sdv/metadata/metadata.py

```diff
@@ -99,3 +99,31 @@ def _convert_to_single_table(self):
             )
 
         return next(iter(self.tables.values()), SingleTableMetadata())
+
+    def set_sequence_index(self, table_name, column_name):
+        """Set the sequence index of a table.
+
+        Args:
+            table_name (str):
+                Name of the table to set the sequence index.
+            column_name (str):
+                Name of the sequence index column.
+        """
+        self._validate_table_exists(table_name)
+        if len(self.tables) > 1:
+            warnings.warn('Sequential modeling is not yet supported on SDV Multi Table models.')
```

Comment:
I think you can get rid of the yet as I don't think there is a plan to add support for that

Reply:
Should we put a different message instead?